src/drivers/power_monitor/ina238/ina238.cpp

```diff
@@ -90,11 +90,10 @@ INA238::INA238(const I2CSPIDriverConfig &config, int battery_index) :
 	_register_cfg[2].clear_bits = ~_shunt_calibration;
 
 	// We need to publish immediately, to guarantee that the first instance of the driver publishes to uORB instance 0
-	_battery.setConnected(false);
-	_battery.updateVoltage(0.f);
-	_battery.updateCurrent(0.f);
-	_battery.updateTemperature(0.f);
+	setConnected(false);
 	_battery.updateAndPublishBatteryStatus(hrt_absolute_time());
+
+	_retries = 5;
```

Comment:
I'd remove this one then:
https://github.com/PX4/PX4-Autopilot/blob/7d509d832acb61e8928ce19c2c73c87c67ea67eb/src/drivers/power_monitor/ina238/ina238.cpp#L184
It's only in INA238 and only on reset and "only" 3 retries compared to 5.